src/vs/workbench/contrib/inlineChat/browser/media/inlineChat.css

```diff
@@ -11,14 +22,30 @@
 	max-width: unset;
 }
 
-.monaco-workbench .inline-chat {
+.monaco-workbench .zone-widget.inline-chat-widget .interactive-session .chat-input-container {
```

Comment:
Fyi you can use CSS nesting now. We downlevel it in our build for old Safari, but it makes authoring much nicer :)